[DOCS] security_risk.md: create a file to discuss about possible security risks

bmarchesini opened the pull request:
Create a file security_risk.md to add consistent informations for the risk of someone having access to your gekko by establishing what can happen and how to prevent that (TODO).

What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Docs update by creating a file about possible risks and how to protect yourself from them.

What is the current behavior? (You can also link to an open issue here)
None information about this subject.

What is the new behavior (if this is a feature change)?
Add consistent information for the risk of someone having access to your gekko by establishing what can happen and add a TODO section to the prevention informations.

Other information:

bmarchesini committed:
[DOCS] create a file to discuss about possible security risks

Create a file security_risk.md to add consistent informations for the risk of someone having access to your gekko by establishing what can happen and how to prevent that (TODO).

askmike commented:
There is no such thing as a "Gekko account".

Can you make it clear that: These risks only apply to people running a gekko server (eg. start the UI). This doesn't apply to CLI usage.

How to secure your server is described here: https://gekko.wizb.it/docs/installation/configuring_gekko_on_a_server.html

These security risks hardly apply to people not running Gekko on a personal computer. Can we make that clear as well? What I mean is that if someone has physical access to your computer DoSsing it via the Gekko API is the last thing they would do. They also don't have to look up API keys stored by Gekkos as they can directly access your exchange accounts and email.

bmarchesini commented:
I do all this as soon as I have time :) I'm very busy at the moment

bmarchesini committed:
[DOCS] security_risk.md: Updating information in accordance with the remarks

The purpose of this section is to describe the risks related to the use of gekko on a server if it is not properly protected, that is why we refer them to another section to configure its server and protect it.

bmarchesini commented:
This is Mike, I made a new commit. I would also like to thank you for all you are teaching me and for your openness to your community. You are always good advice and it makes you want to get involved in such projects!
